### ISSUE TITLE
[BUG] [0.9.1] Cannot reliably add class features to spawned tokens of NPCs

### ISSUE THREAD

**Issue by VexWerewolf** [Eranziel/foundryvtt-lancer#175]
**Describe the bug**
Once a token version of an NPC is spawned in a scene, attempting to add class or template features to its feature section will work the first time, but then will invariably fail every single time afterwards, producing the following error:

**Cannot read propery 'items' of null**

**To Reproduce**
1. Create a test NPC (in this example, an Ultra Commander Mirage).
2. Create a test map
3. Spawn a token of the test NPC on the test map by dragging and dropping.
4. Open token configuration and ensure that under "Character," "Link Actor Data" is OFF.
5. Double-click the token, opening its sheet.
6. Open the NPC Feature list in the Compendium.
7. Drag and drop a feature from the list onto the token's sheet. This should work the first time you do it.
8. Drag and drop another feature from the list onto the token's sheet. This should begin producing errors.

**Expected behavior**
In 0.1.20, token versions of NPCs that were not linked to their master sheet could have class features added to them without incident.

**Screenshots**
![NPC token class features](https://user-images.githubusercontent.com/83786280/117373410-1343d380-aec3-11eb-9944-9fa7f1cfde46.jpg)

**Platform (please complete the following information):**
 - OS: Windows 10
 - Foundry version 0.7.9
 - Lancer system version 0.9.1

**Discord contact info**
Vex#1075

**Comment by ScyllPoesis** [comment]
Tracked down the error to NPC sheet handling of the drop failing at this line, `npc-sheet.ts:199`:
```
let new_live_item = await item_mm.insinuate(this_inv, new_ctx);
```

**Comment by ScyllPoesis** [comment]
I've noticed that there are a few situations I've run into where remaking the token from source fixes a lot of my errors, so good to know. It's probably best practice for now to make copies of NPC sheets when editing and keep tokens unmodified as much as possible.

**Comment by VexWerewolf** [comment]
@ScyllPoesis @whitespine Upon further testing I've discovered that tokens not linked to sheets also don't reliably hold status markers and they also don't update their resource bars reliably.

**Comment by whitespine** [comment]
(Almost) any updates to the data model will fail once the token actor enters this invalid state. I imagine its some sort of server side corruption, though it is impossible to tell.

**Comment by ScyllPoesis** [comment]
Presumably this stuff will be resolved thanks to the new way Foundry handles unlinked tokens and their backing actors, so we'll revisit this after fvtt8 migration.

**Comment by whitespine** [comment]
> Is there a way of bug-reporting to Foundry itself?

There's feedback channels on the foundry discord. However ATM they're all for the 0.8 pre-release so idk if a .7 problem will get much traction

**Comment by Grygon** [comment]
Confirmed working with FVTT 0.8

**Comment by Grygon** [comment]
Closing - Fixed in 0.9.3
